### DISCUSSION TITLE
Is this intentional?

### DISCUSSION THREAD

**Discussion by CaptGBread** [soulhotel/FF-ULTIMA#173] [Q&A]
Hello! I really appreciate your work, time and effort for this theme, love it. So I have some questions about the autohide on vertical tabs. I need to click this sidebar button on top of firefox in order for the autohide to work. Is this intentional? It's not a big deal really, but this button resets itself after firefox is closed, so I need to click this button everytime I open firefox. Is it just another firefox behaviour? Is there a way to keep this button 'clicked' even after I close firefox so I dont need to click it the next time I open it? On the brightside tho, I feel the autohide animation is smoother than ever before. 

Im on Windows 11 using Firefox 132.0 with the latest FF Ultima 1.9.1

p.s thank you for keeping this theme up to date. 

https://github.com/user-attachments/assets/f82068fc-412c-4553-89f2-3e1407f6e2d0

**Comment by soulhotel** [comment]
Not intentional, its Firefox behavior,

basically the vertical tabs sidebar has a collapsed state (icons) and an expanded state (everything). And that button is a toggle between the two.

As soon as the "second" sidebar is triggered (if you press the history sidebar or bookmarks, sidebery, etc) then the vertical tabs sidebar will be forced to collapse. And the auto-hiding only works when expanded (obviously to see the tab label, close button, text, everything..)

I found that if you trigger the second sidebar for like bookmarks or sidebery or whatever, then you should press that same bookmark or sidebery button (instead of the "sidebar button"). This way the vertical tabs sidebar will automatically return to the expanded/autohiding state. It is pretty annoying, but I haven't had to touch the button for more than a week now by doing it this way.


https://github.com/user-attachments/assets/72a5316e-fc08-4ed0-b839-be504470af99



Also for closing the window, maybe this is a 132 issue but on 133 it does remember if its in an expanded/collapsed state - on close. I'll look into that more tomorrow to see what that could be.

**Reply by roadbloack** [reply]
hey, just wanted to ask, how did you change the bottom of the vertical tabs, I want that as well
![image](https://github.com/user-attachments/assets/7ed5725a-a9ec-476c-8025-2f1b4f908b0d)

**Reply by soulhotel** [reply]
Honestly this is annoying for me, I did nothing here. This is how it looks on Developer Edition (version 134 right now).

On default release (133) and Nightly (135) the toolbar buttons are Vertical like you have them... I wish the Firefox Devs would stick to the Horizontal toolbar, it just looks so much better. And Idk why they have it different across Firefox Editions.

**Reply by roadbloack** [reply]
ok thanks

**Comment by CaptGBread** [comment]
No longer an Issue in Firefox 133. Thanks! Keep up the great work!

**Reply by soulhotel** [reply]
Awesome! Np.
